### PR TITLE
PB Banner UI test

### PR DESCRIPTION
**Pull request by dpatil-magento**
## Description

TODO: Describe your changes in detail here.

## Related Issue

Closes https://jira.corp.magento.com/browse/PWA-1540.

### Verification Steps

1. Cypress `banner.spec.js` should pass
Steps - 
Go to venia-integration-test dir
Run `yarn install`
Run `yarn test`
Run `banner.spec.js` from cypress ui

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: venia-integration-tests/src/tests/pageBuilder/integrationTests/banner.spec.js
Comment (davemacaulay) [inline venia-integration-tests/src/tests/pageBuilder/integrationTests/banner.spec.js:99]: Is there a way to avoid these high timeouts? and rather wait for a piece of content we expect to render near the end of the content?

Comment (dpatil-magento) [reply venia-integration-tests/src/tests/pageBuilder/integrationTests/banner.spec.js:99]: This is max time given to plugin to process screenshot updates/comparison and not related to page load. Also it does not actually wait 60 secs every time, as soon as screen comparison is done control does move to next step.

Comment (davemacaulay) [reply venia-integration-tests/src/tests/pageBuilder/integrationTests/banner.spec.js:99]: Ah super cool, thanks for clarifying.

File: venia-integration-tests/src/actions/accountAccess/index.js
Comment (revanth0212) [inline venia-integration-tests/src/actions/accountAccess/index.js:1]: Looks like you'll have to update this PR after the wishlist PR merge.

File: venia-integration-tests/src/tests/pageBuilder/integrationTests/banner.spec.js
Comment (eug123) [inline venia-integration-tests/src/tests/pageBuilder/integrationTests/banner.spec.js:4]: Seems like these imports not needed

Comment (dpatil-magento) [reply venia-integration-tests/src/tests/pageBuilder/integrationTests/banner.spec.js:2]: Updated as needed, thanks :)

File: venia-integration-tests/src/tests/pageBuilder/integrationTests/banner.spec.js
Comment (revanth0212) [inline venia-integration-tests/src/tests/pageBuilder/integrationTests/banner.spec.js:17]:
```suggestion
    });
    
```

File: venia-integration-tests/src/actions/routes/index.js
Comment (revanth0212) [inline venia-integration-tests/src/actions/routes/index.js:1]: I dont think we need this any more. We moved these to custom Cypress actions in the wishlist PR.

File: venia-integration-tests/src/actions/accountAccess/index.js
Comment (revanth0212) [inline venia-integration-tests/src/actions/accountAccess/index.js:28]: I dont think we need these any more. We moved these to custom cypress actions. Or am I missing something?

File: venia-integration-tests/src/actions/accountAccess/index.js
Comment (revanth0212) [inline venia-integration-tests/src/actions/accountAccess/index.js:57]: Please move assertions to assertions folder.

File: venia-integration-tests/src/actions/wishlist/index.js
Comment (revanth0212) [inline venia-integration-tests/src/actions/wishlist/index.js:17]: Please move assertions to the assertions folder.

**Review by revanth0212 — CHANGES_REQUESTED** [review]
Tests look fine but some clean-up is needed on the assertions end.

File: venia-integration-tests/src/actions/wishlist/index.js
Comment (dpatil-magento) [reply venia-integration-tests/src/actions/wishlist/index.js]: Dont know how I ended up here but updated all, thanks. Please re-check :)

**Review by revanth0212 — APPROVED** [review]

**Comment by sirugh** [comment]
Getting test failures for 1-3 - seems the snaps were saved at 1200 width but the test runner is now outputting 1440, at least on my machine.

**Comment by sirugh** [comment]
Will merge after checks pass! Re-verified, had to clear local cypress chrome cache.

**Pull request merged**
